everviz/plugins/_crossplot_plugin/crossplot_plugin.py

```diff
@@ -0,0 +1,82 @@
+from uuid import uuid4
+import numpy as np
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+from webviz_config import WebvizPluginABC
+
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_layout import (
+    _get_layout,
+)
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_callback import (
+    _crossplot_callback,
+)
+from everviz.data.load_csv._get_data import get_data
+from everviz.plugins._crossplot_plugin.crossplot_control_group_plugin import _get_trace
```

Comment:
I am not sure this import is used and it is a bit strange because there is already a `_get_trace()` defined in this file.